GoogleForCreators/web-stories-wp
# History: reduce polluting history

## Context


## Summary
Ignores various media properties that change in the background, polluting history.

Note: There are still things that pollute history but it takes quite a bit of debugging to figure those out so we should probably create follow-up investigation issues per case.


## Relevant Technical Choices



## To-do

- [x] More testing
- ~Try to track down values still causing extra entries in the history~ This takes quite a bit of time, it might be good to create separate issues for separate cases for further investigation, e.g. "Dragging video onto the page from local files".



## User-facing changes



## Testing Instructions





This PR can be tested by following these steps:

This PR is not very straightforward to test but can be tested comparing `main` and the results on this PR in the following way:
1. Reload the page on `main`
2. Do an action (wait until it has fully finished if there's processing happening)
3. Click "Undo" until getting back to the initial state
4. Log how many times you had to click "Undo"
5. Compare with results of the same action for this PR, verify that it takes less steps on this PR than on `main`. Would be great to log the differences in GitHub as well.

The actions to test:
- Removing audio from a video
- Dragging a video from local directly onto the Page
- Trimming a video

## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #10009
